index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css"/>
+  <title>Portfolio</title>
+</head>
+<body>
+  <main class="container">
+  
+    <header class="header">
+      <ul>
+        <li><img src="./assets/Time.png" alt="clock" class="top"></li>
+        <li><img src="./assets/Group 3.png" alt="bars" class="top"></li>
+      </ul>
+    </header>
+    <nav>
+      <a href="#" target="_parent" class="logo">My Portfolio</a>
+      <img src="./assets/Union.png" class="mobile_menu_icon"/>
+    </nav>
+
+    <section>
+      <div class="hero">
+        <p class="author">I’m Tanveer Glad to see you!</p>
```

Comment:
- Please, let's use the h1 tag here as this is the main heading of your portfolio.